metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala

```diff
@@ -432,7 +442,7 @@ object ServerCommands {
     "Stop Ammonite build server"
   )
 
-  def all: List[Command] =
+  def all: List[BaseCommand] =
```

Comment:
I'd turn `all` into `Map[String, BaseCommand]`, preferably with `String` replaced by some new type like `CommandId`

Reply:
I think I will leave as is, I don't see a clear benefit for it and that would be a few changes to make it work.